modules/publicize/publicize.php

```diff
@@ -418,6 +419,91 @@ function save_meta( $post_id, $post ) {
 		// Next up will be ::publicize_post()
 	}
 
+	public function update_published_message( $messages ) {
+		global $post_type, $post_type_object, $post;
+		if ( ! $this->post_type_is_publicizeable( $post_type ) ) {
+			return $messages;
+		}
+		$view_post_link_html = '';
+		$viewable = is_post_type_viewable( $post_type_object );
+		if ( $viewable ) {
+			$view_text = __( 'View post' ); // intentinally omitted domain
+
+			if ( $post_type == 'jetpack-portfolio' ) {
+				$view_text = esc_html__( 'View project', 'jetpack' );
+			}
+
+			$view_post_link_html = sprintf( ' <a href="%1$s">%2$s</a>',
+				esc_url( get_permalink( $post ) ),
+				$view_text
+			);
+		}
+
+		$services = $this->get_publicizing_services( $post->ID );
+		if( empty( $services ) ) {
+			return $messages;
+		}
+
+		$labels = array();
+		foreach( $services as $service ) {
+			$labels[] = sprintf(
+				_x( '%1$s: %2$s', 'Service: Account publicizing to', 'jetpack' ),
```

Comment:
I think we would benefit from a translator comment here to make it clear what 1 and 2 are